include/hw_isolation_event/hw_status_manager.hpp

```diff
@@ -84,6 +84,13 @@ class Manager
      */
     std::vector<std::string> _requiredHwsPdbgClass;
 
+    /**
```

Comment:
Host  "Quiesced" can happen after "SystemInitComplete"  status also . is this required?

Reply:
It will be covered separately, right? since these D-Bus properties are hosted at different interfaces so if in any case these properties got changed and we are interested to handle it, we will pull the deconfig reason, right?